pkg/services/team/model_test.go

```diff
@@ -0,0 +1,49 @@
+package team
+
+import (
+	"encoding/json"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestTeamConversion(t *testing.T) {
+	src := Team{
+		ID:      123,
+		UID:     "abc",
+		Name:    "TeamA",
+		Email:   "team@a.org",
+		OrgID:   11,
+		Created: time.UnixMilli(946713600000).UTC(),  // 2000-01-01
+		Updated: time.UnixMilli(1262332800000).UTC(), // 2010-01-01
+	}
+
+	dst := src.ToResource()
+
+	require.Equal(t, src.Name, dst.Spec.Name)
+
+	out, err := json.MarshalIndent(dst, "", "  ")
+	require.NoError(t, err)
+	//fmt.Printf("%s", string(out))
+	require.JSONEq(t, `{
+		"metadata": {
+		  "createdBy": "",
```

Comment:
😬 but at least we are getting closer to a k8s object -- this will require some work!